### PR TITLE
created navbarbutton

### PR DESCRIPTION
**Pull request by nur221282**
gh pr checkout 4

### CODE REVIEW

**Comment by nur221282** [comment]
gh pr checkout 4

File: src/components/navbarbutton.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React,{useState} from 'react'
+import Select, { Option } from 'react-button-navbar';
+ 
+
+
+    function App(){
```
Comment (maksim-tsvetkov) [inline src/components/navbarbutton.tsx:6]: Hmm, extra tabs? Please align the function to left.

Comment (nur221282) [reply src/components/navbarbutton.tsx:6]: Can I add this code?
headline: {
    textAlign: 'center', // <-- the magic
    fontWeight: 'bold',
    fontSize: 18,
    marginTop: 0,
    width: 200,
    backgroundColor: 'brown',
  }

**Review by nur221282 — COMMENTED** [review]
https://immutable.atlassian.net/browse/II-74

File: src/components/navbarbutton.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React,{Fragment, useState} from 'react'
+import Select, { Option } from 'react-button-navbar';
+ 
+
+
+    function NavbarButton(){
```
Comment (pano-skylakis) [inline src/components/navbarbutton.tsx:6]: There are unnecessary spaces on the left of this whole function that can be removed

File: src/components/navbarbutton.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        const [current, setCurrent] = useState("opt1")
+        return (
+        <><button>CONNECT</button>
```
Comment (pano-skylakis) [inline src/components/navbarbutton.tsx:9]: This <button> should be on a new line

**Review by pano-skylakis — COMMENTED** [review]
You also have merge conflicts that need to be resolved as well - might be a good idea to work through them with your team :)

File: src/components/navbarbutton.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React,{Fragment, useState} from 'react'
+import Select, { Option } from 'react-button-navbar';
+ 
+
+
+    function NavbarButton(){
+        const [current, setCurrent] = useState("opt1")
```
Comment (pano-skylakis) [inline src/components/navbarbutton.tsx:7]: This doesn't look like it's being used anywhere

File: src/components/navbarbutton.tsx (hunk 2 above)
Comment (nur221282) [reply src/components/navbarbutton.tsx:6]: Thank you. Done!

File: src/components/navbarbutton.tsx (hunk 3 above)
Comment (nur221282) [reply src/components/navbarbutton.tsx:9]: Thank you. Done!

File: src/components/navbarbutton.tsx (hunk 4 above)
Comment (nur221282) [reply src/components/navbarbutton.tsx:7]: I deleted it. Thank you ☺️

**Review by maksim-tsvetkov — COMMENTED** [review]
One conflict left, aside from that, all looks good 👍

**Comment by nur221282** [comment]
> 

Trying to fix it, thanks Maksim :)

**Review by nur221282 — COMMENTED** [review]
Duplicated files had been deleted.

**Comment by nur221282** [comment]
Duplicated files had been deleted.

**Comment by nur221282** [comment]
Dear mentors could you please review? Duplicated files had been deleted.

**Comment by silvamarcel** [comment]
Why did you close this PR?

**Comment by nur221282** [comment]
After deleting duplicates, I have created #PR16.

**Comment by nur221282** [comment]
It has been deleted duplicate files, and I created a new branch with a clear view.
--
Sent from Mail.ru app for Android Thursday, 14 July 2022, 11:42am +10:00 from Marcel de Jesus Silva  ***@***.*** :

**Pull request closed without merging**